.github/workflows/jruby.yml

```diff
@@ -0,0 +1,31 @@
+name: build
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '12 3 * * *'
+
+jobs:
+  build:
+    name: build (${{ matrix.ruby }} / ${{ matrix.os }})
+    strategy:
+      matrix:
+        ruby: [ 'jruby-9.3' ]
+        os: [ ubuntu-latest ]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1.81.0
+      with:
+        ruby-version: ${{ matrix.ruby }}
+        bundler-cache: true # 'bundle install' and enable caching
+    - name: Install dependencies
+      run: bundle install
```

Comment:
```suggestion
```

Now, things are cached. 

Also: you will want to use `bundle exec` for the rake tasks, too.

Reply:
Ok, made those changes and updated setup-ruby to pick up 9.3 properly.